HAL_extern.h

```diff
@@ -153,7 +152,6 @@ void    initConfKey(void) {
 
 #ifndef USE_OWN_ISR_PCINT2_vect
 	ISR (PCINT2_vect) {
-		pcInt[2].prev = pcInt[2].cur;
```

Comment:
Das ist wohl richtig.
Ich würde hier eh noch mal ran gehen, denn die PCINT ISR's sind in dieser Form nur eingeschränkt verwendbar. Daher auch das #ifndef USE_OWN_ISR_PCINT0_vect darum, so dass man seine eigenen ISR's definieren kann.
Dann merge ich das mal. Kompiliert hat es schon mal.